java/code/src/com/suse/manager/reactor/messaging/RegisterMinionEventMessageAction.java

```diff
@@ -397,6 +397,18 @@ private void reactivateSystem(String minionId, String machineId, String reActiva
         });
     }
 
+    private void setMinionName(String minionId, MinionServer minion, Optional<ValueMap> grains) {
+        if (grains.isPresent()) {
+            minion.setName(
+                    grains.get().getMap("susemanager")
+                            .flatMap(s -> s.getOptionalAsString("profile_name"))
+                            .orElse(minionId));
+        }
+        else {
+            minion.setName(minionId);
+        }
```

Comment:
This is only a nitpick, but we are mixing styles here.
What about:
```suggestion
        grains.ifPresentOrElse(
                grain -> {
                    minion.setName(
                        grain.getMap("susemanager")
                            .flatMap(s -> s.getOptionalAsString("profile_name"))
                            .orElse(minionId));
                },
                () -> minion.setName(minionId)
        );
```

Reply:
I agree. Suggestion taken in commit squash